### DISCUSSION TITLE
打包成功了但是 Artifacts 没有显示是为什么？

### DISCUSSION THREAD

**Discussion by iibrat** [tw93/Pake#649] [Q&A]
![微信图片_20240120141653](https://github.com/tw93/Pake/assets/108867675/a321628e-9cd4-487d-894c-ff497f310db8)
打包成功了但是 Artifacts 没有显示是为什么？

**Comment by tw93** [comment]
<img width="1059" alt="image" src="https://github.com/tw93/Pake/assets/8736212/cfd21cad-f322-46ab-a2d4-e5e5aaac896d">
退到上一个页面，在这里
